front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -19,14 +18,58 @@
     }
   }
 
+  .micro-macro-buttons {
+    button {
+      color: var(--grey60);
+    }
+    font-size: 0.875rem;
+    margin: 1.813rem 0 2.063rem 0;
+    padding: 0.25rem 0;
+    display: inline-flex;
+    background-color: var(--ambiantB10);
+    border-radius: 0.375rem;
+    box-shadow:
+      0px 2px 2px -1px rgba(255, 171, 88, 0.27),
+      0px 1px 2px rgba(0, 0, 0, 0.16);
+    .micro-button {
+      border-right: solid 0.5px var(--grey20);
+      padding: 0.188rem 0.625rem 0.188rem 1rem;
+    }
+    .macro-button {
+      border-left: solid 0.5px var(--grey20);
+      padding: 0.188rem 1rem 0.188rem 0.625rem;
+    }
+    .micro-button:hover {
+      background-color: var(--white);
+      box-shadow:
+        0 0.25rem 0rem var(--white),
+        0 -0.25rem 0rem var(--white);
+      border-radius: 0.375rem 0 0 0.375rem;
+      color: var(--black);
+    }
+    .macro-button:hover {
+      background-color: var(--white);
+      box-shadow:
+        0 0.25rem 0rem var(--white),
+        0 -0.25rem 0rem var(--white);
+      border-radius: 0 0.375rem 0.375rem 0;
+      color: var(--black);
+    }
```

Comment:
Didn't find these styles on the mockup. Are these requests from @thibautsailly ?

![image](https://github.com/user-attachments/assets/229c21ca-2868-482c-90ff-96f6f0a6cea6)

Reply:
I just asked him for some details
Mockup: 
![image](https://github.com/user-attachments/assets/5877e5dc-83c0-4323-9cf3-65fe90abeaef)

Reply:
Done

Reply:
Can't we have access to this mockup as well ? :x